pennylane/_grad.py

```diff
@@ -63,42 +62,23 @@ class grad:
             to determine differentiability, by examining the ``requires_grad``
             property.
 
-        method (str): Specifies the gradient method when used with
```

Comment:
I'm a little torn about removing these from docstrings, but I'll leave it up to you/other reviewers

Reply:
Yep, how come this is being removed? Please leave this in, as we should be documenting all keyword arguments of a function!